docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+| constructor (class)     | `ref`              | Only classes can have different constructor capabilities                |
+| receiver (function)     | `box`              | Functions cannot mutate class state by default (use `func ref`)         |
+| receiver (behaviour)    | `ref`              | Actors can mutate their state from inside a behaviour                   |
+| recover (immutable)     | `val`              | An immutable value (box or val) is recovered to `val` by default        |
+| recover (mutable)       | `iso`              | An immutable value (box or val) is recovered to `val` by default        |
+| literal (object)        | `ref`              | object literals default to same capabilities as their non-literal types |
+| literal (actor)         | `tag`              | object literals default to same capabilities as their non-literal types |
+| literal (primitive)     | `val`              | object literals default to same capabilities as their non-literal types |
+| lambda (no captures)    | `val`              | lambda capabilities depend on whether they capture their environment    |
+| lambda (captures)       | `ref`              | lambda capabilities depend on whether they capture their environment    |
+
+## Primitive default capabilities
+
+Let's start with primitives because they're easy. All primitives have `val` capabilities. They are immutable, they are read only, there is only one of each primitive, and they are accessible from all actors. So `val`.
+
+In fact, defining a primitive as `primitive val MyPrimitive` is a syntax error. Primitives are so `val` that you can't even tell them they are `val`.
+
+## Actor default capabilities
+
+Similar to primitives, actors always have a `tag` default capability. Like primitives, it is not possible to specify a different default capability for tags.
+
+```pony
+actor Actor
+```
+
+has `tag` capabilities
+
+```pony
+actor ref Actor
+```
+
+is illegal
+
+## Class default capabilities
+
+By default, a class has `ref` capability. Therefore, these two class definitions are equivalent:
+
+```pony
+class ref RefClass
+    ...
+
+class RefClass
+    ...
+```
+
+But whether specified by default or explicitly as `class ref` (or `class val` or any other capability) what does it actually mean? A class or actor's default capability only applies when you _define_ a field or local type. This can happen either as `let` or `var`, or when specifying a parameter on a function or behaviour.
+
+## Field, Variable, and Parameter capabilities
+
+For example, `String` is defined in the standard library as `class val String`. So it's default capability is the immutable shareable `val` capability. That means that all the `String`s in the following actor definition could be replaced with `String val`:
+
+```pony
+actor UsesStrings
+    let s1: String
+    var s2: String
+
+    new create(s1': String, s2': String) =>
+        s1 = s1'
+        s2 = s2'
+
+    be do_thing(s3: String) =>
+        None
+
+```
```

Comment:
```suggestion
```pony
actor UsesStrings
    let s1: String
    var s2: String

    new create(s1': String, s2': String) =>
        s1 = s1'
        s2 = s2'

    be do_thing(s3: String) =>
        None
```
```

Reply:
I think github made this suggestion weird and it was supposed to just be deleting the extra newline. If I missed another change in it, let me know.